libgraph/include/katana/PropertyGraph.h

```diff
@@ -870,6 +870,10 @@ class KATANA_EXPORT PropertyGraph {
     return node_iterator(node_id);
   }
 
+  Result<bool> IsNodeIndexPresent(const std::string& column_name);
```

Comment:
Isn't this a duplicate for https://github.com/KatanaGraph/katana/blob/36253df6ce8f334b3c4e617a132044eb966091c3/libgraph/include/katana/PropertyGraph.h#L902